DUI3-DX/Connectors/Rhino/Speckle.Connectors.Rhino7/Bindings/RhinoSendBinding.cs

```diff
@@ -49,29 +38,32 @@ public class RhinoSendBinding : ISendBinding, ICancelable
   /// <summary>
   /// Keeps track of previously converted objects as a dictionary of (applicationId, object reference).
   /// </summary>
-  private readonly Dictionary<string, ObjectReference> _convertedObjectReferences = new();
+  //private readonly Dictionary<string, ObjectReference> _convertedObjectReferences = new();
 
   public RhinoSendBinding(
     DocumentModelStore store,
     RhinoIdleManager idleManager,
-    RhinoSettings rhinoSettings,
     IBridge parent,
     IBasicConnectorBinding basicConnectorBinding,
-    IScopedFactory<ISpeckleConverterToSpeckle> speckleConverterToSpeckleFactory,
-    RhinoContext rhinoContext
+    IEnumerable<ISendFilter> sendFilters,
+    Func<SendOperation> sendOperationFactory,
+    SendBindingUICommands.Factory sendBindingFactory,
```

Comment:
Do we need this to be a factory?

Reply:
You were right, send operation did not need to be a factory. However, dependencies down the line do kind of need to be factories. What would you use instead of Func?